dashboards/aws-feeds/aws-feeds.yaml

```diff
@@ -0,0 +1,2828 @@
+          RelationalTable:
+            DataSourceArn: ${athena_datasource_arn}
+            Catalog: AwsDataCatalog
+            Schema: ${athena_database_name}
+            Name: aws_feeds_blog_post
+            InputColumns:
+            - Name: title
+              Type: STRING
+            - Name: link
+              Type: STRING
+            - Name: description
+              Type: STRING
+            - Name: date
+              Type: STRING
+            - Name: creator
+              Type: STRING
+            - Name: category
+              Type: STRING
+            - Name: year
+              Type: STRING
+            - Name: month
+              Type: STRING
+            - Name: day
+              Type: STRING
+      LogicalTableMap:
+        633c14f6-0bff-4f6a-ae1b-6508d3a1781f:
+          Alias: aws_feeds_blog_post
+          DataTransforms:
+          - CastColumnTypeOperation:
+              ColumnName: date
+              NewColumnType: DATETIME
+              Format: YYYY-MM-dd'T'HH:mm:ss'Z'
+          - ProjectOperation:
+              ProjectedColumns:
+              - title
+              - link
+              - description
+              - date
+              - creator
+              - category
+              - year
+              - month
+              - day
+          Source:
+            PhysicalTableId: ce6e7be8-3a44-4325-b55d-26ab590920d1
+      ImportMode: SPICE
+    dependsOn:
+      views:
+      - aws_feeds_blog_post
+    schedules:
+    - default
+  aws_feeds:
+    data:
+      DataSetId: d50473c6-53c1-45f5-9185-361136fb02ef
+      Name: aws_feeds
+      PhysicalTableMap:
+        6593bf38-e15a-41d7-b85c-9d5d2ea4027f:
+          RelationalTable:
+            DataSourceArn: ${athena_datasource_arn}
+            Catalog: AwsDataCatalog
+            Schema: ${athena_database_name}
+            Name: aws_feeds
+            InputColumns:
+            - Name: published
+              Type: STRING
+            - Name: feed_type
+              Type: STRING
+            - Name: title
+              Type: STRING
+            - Name: url
+              Type: STRING
+            - Name: category
+              Type: STRING
+      LogicalTableMap:
+        13d7470d-f377-4f2a-bfc5-cda8cb6f493a:
+          Alias: aws_feeds
+          DataTransforms:
+          - CastColumnTypeOperation:
+              ColumnName: published
+              NewColumnType: DATETIME
+              Format: YYYY-MM-dd'T'HH:mm:ss'Z'
+          - ProjectOperation:
+              ProjectedColumns:
+              - published
+              - feed_type
+              - title
+              - url
+              - category
+          Source:
+            PhysicalTableId: 6593bf38-e15a-41d7-b85c-9d5d2ea4027f
+      ImportMode: SPICE
+    dependsOn:
+      views:
+      - aws_feeds
+    schedules:
+    - default
+crawlers: {}
+views:
+  aws_feeds:
+    data: |-
+      CREATE OR REPLACE VIEW ${athena_database_name}.aws_feeds AS
```

Comment:
Any reason why not to create it in ${data_collection_database_name} ?

Reply:
You are right, I have removed the param as being duplicated.

Reply:
The reason is to align with other dashboards like extended support and others that use cur and data collection 

I would recommend keeping 2 databases. They will be autodetected soon.

Reply:
I don't see any benefit of having them align, but if you think this will help I'm fine